Runtime/Native/Windows/NativeClient.cs

```diff
@@ -249,50 +249,67 @@ public void SetAttribute(string key, string value)
         /// </summary>
         public IEnumerator SendMinidumpOnStartup(ICollection<string> clientAttachments, IBacktraceApi backtraceApi)
         {
-            // Path to the native crash directory
-            string nativeCrashesDir = Path.Combine(
-                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
-                    string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName));
+            // Path to the native crash directories
+            string tempDirectory = string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName);
+            string[] crashDirectories = new string[2] {
+                Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), tempDirectory),
+                Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), tempDirectory)
+            };
+              
+            List<string> nativeCrashesDirs = new List<string>();
```

Comment:
```suggestion
            var nativeCrashesDirs = new List<string>();
```

You can use var here